installer/conf/sudoers

```diff
@@ -34,6 +34,9 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 # Enable nxOMSAuditdResource
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh
 
+#Enable shutdown for update management
+omsagent ALL=(ALL) NOPASSWD: /sbin/shutdown -r *
```

Comment:
there is no delay option explicitly, I meant now or we can add a min number as delay.

so it can be "shutdown -r 6" for 6mins from current time